### ISSUE TITLE
Dont work with css-loader 4.0.0

### ISSUE THREAD

**Issue by tarnishablec** [gajus/to-string-loader#19]
```ts
import raw from '!!to-string-loader!css-loader!sass-loader!./index.scss'
```

output a [object Module]

**Comment by JonWallsten** [comment]
I can add that it's because the css-loader 4.x.x is using `{ esModule: true }` as default.

**Comment by tarnishablec** [comment]
> I can add that it's because the css-loader 4.x.x is using `{ esModule: true }` as default.

**Comment by 3cp** [comment]
Fixed by #20

**Comment by brunolemos** [comment]
Current workaround: `require('!!to-string-loader!css-loader?esModule=false!sass-loader!./index.scss')`
